tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html

```diff
@@ -75,11 +80,18 @@
         white-space: nowrap;
         text-transform: capitalize;
       }
+      div.recommendationDetails {
+        color:green;
+      }
+      div.recommendationDetails a{
```

Comment:
nit: Lets place a space after each selector (before the opening curly brace).

Reply:
Done